release-doc-in-process/all_vf.TTL

```diff
@@ -323,19 +323,57 @@ vf:containedIn a            owl:ObjectProperty ;
         rdfs:comment        "Used when a stock economic resource contains units also defined as economic resources." .
 
 vf:hasBeginning a           owl:DatatypeProperty ;
-        rdfs:domain         time:TemporalEntity ;
+        rdfs:domain         [ owl:unionOf (
+                                time:TemporalEntity
+                                vf:Commitment
+                                vf:EconomicEvent
+                                vf:Intent
+                                vf:Proposal
+                                vf:Scenario
+                                vf:Process) ] ;
         rdfs:range          xsd:dateTimeStamp ;
         vs:term_status      "unstable" ;
         owl:propertyChainAxiom (time:hasBeginning time:inXSDDateTimeStamp) ;
         rdfs:comment        "The planned or actual beginning of a flow or process." .
 
 vf:hasEnd a                 owl:DatatypeProperty ;
-        rdfs:domain         time:TemporalEntity ;
+        rdfs:domain         [ owl:unionOf (
+                                time:TemporalEntity
+                                vf:Commitment
+                                vf:EconomicEvent
+                                vf:Intent
+                                vf:Proposal
+                                vf:Scenario
+                                vf:Process) ] ;
         rdfs:range          xsd:dateTimeStamp ;
         vs:term_status      "unstable" ;
         owl:propertyChainAxiom (time:hasEnd time:inXSDDateTimeStamp) ;
         rdfs:comment        "The planned or actual end of a flow or process." .
 
+vf:hasPointInTime a         owl:DatatypeProperty ;
+        rdfs:domain         [ owl:unionOf (
+                                time:Instant
+                                vf:Commitment
+                                vf:EconomicEvent
+                                vf:Intent) ] ;
+        rdfs:range          xsd:dateTimeStamp ;
+        vs:term_status      "unstable" ;
+        owl:equivalentProperty  time:inXSDDateTimeStamp ;
```

Comment:
Should remove this line and replace with:
---
owl:propertyChainAxiom (time:hasEnd time:inXSDDateTimeStamp) ;

Reply:
I wasn't sure the best way to do it.  My thinking was that if we can use hasBeginning and hasEnd to interface directly with owl time, and if we specify that those have to be returned if hasPointInTime is what was entered, then we could be good with either hasPointInTime or inXSDDateTimeStamp.  But I'm the first to admit I don't know what I'm doing on this one technically.

Reply:
done